tests/kafkatest/services/monitor/jmx.py

```diff
@@ -27,7 +27,8 @@ class JmxMixin(object):
     - we assume the service using JmxMixin also uses KafkaPathResolverMixin
     - this uses the --wait option for JmxTool, so the list of object names must be explicit; no patterns are permitted
     """
-    def __init__(self, num_nodes, jmx_object_names=None, jmx_attributes=None, jmx_poll_ms=1000, root="/mnt"):
+    def __init__(self, num_nodes, jmx_enabled=True, jmx_object_names=None, jmx_attributes=None, jmx_poll_ms=1000, root="/mnt"):
```

Comment:
I wonder if it would be better to rely on the `JmxTool` wrapper that @mumrah added here: https://github.com/apache/kafka/blob/trunk/tests/kafkatest/tests/core/fetch_from_follower_test.py#L31. We can pull this up into a more convenient location and change console consumer to only construct an instance if `jmx_enabled` is set. That seems a little cleaner than supporting an option to turn off the mixin.

Reply:
makes sense!